wire-ios-data-model/Source/Model/Conversation/ZMConversation+MLS.swift

```diff
@@ -173,24 +173,21 @@ public extension ZMConversation {
             ]
         )
 
-        let isMLSConveration = NSPredicate(
-            format: "%K == %i && %K != nil",
-            argumentArray: [
-                Self.messageProtocolKey,
-                MessageProtocol.mls.rawValue,
-                Self.mlsGroupIdKey
-            ]
-        )
-
         request.predicate = NSCompoundPredicate(andPredicateWithSubpredicates: [
-            isSelfConversation, isMLSConveration
+            isSelfConversation, .isMLSConversation
```

Comment:
can we do the same for isSelfConversation ?

Reply:
done ✅